src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqExpectationsRepository.kt

```diff
@@ -4,27 +4,30 @@ import org.jooq.DSLContext
 import org.jooq.Record2
 import org.vaccineimpact.api.app.errors.UnknownObjectError
 import org.vaccineimpact.api.app.repositories.ExpectationsRepository
+import org.vaccineimpact.api.app.repositories.ResponsibilitiesRepository
 import org.vaccineimpact.api.db.Tables.*
 import org.vaccineimpact.api.db.fromJoinPath
 import org.vaccineimpact.api.db.tables.BurdenEstimateCountryExpectation
 import org.vaccineimpact.api.db.tables.BurdenEstimateExpectation
 import org.vaccineimpact.api.db.tables.BurdenEstimateOutcomeExpectation
 import org.vaccineimpact.api.db.tables.records.BurdenEstimateExpectationRecord
-import org.vaccineimpact.api.models.CohortRestriction
-import org.vaccineimpact.api.models.Country
-import org.vaccineimpact.api.models.Expectations
-import org.vaccineimpact.api.models.YearRange
+import org.vaccineimpact.api.models.*
 
-class JooqExpectationsRepository(dsl: DSLContext) : JooqRepository(dsl), ExpectationsRepository
+class JooqExpectationsRepository(dsl: DSLContext,
+                                 private val responsibilitiesRepository: ResponsibilitiesRepository)
```

Comment:
Every time I touch the repos I feel keen to refactor; maybe we could bring in the ticket to introduce a logic layer into this sprint?